packages/core-api/src/lib/AuthSessionManager/SamlAuthSessionStore.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { SamlAuthSessionManager, GetSessionOptions } from './types';
+
+type Options<T> = {
+  manager: SamlAuthSessionManager<T>;
+  storageKey: string;
+};
+
+export class SamlAuthSessionStore<T> implements SamlAuthSessionManager<T> {
```

Comment:
Same thing here, is this addition needed? How is it different from the existing session store?

Reply:
Yeah, I thought at first that it is going to be different, but then I think can reuse the generic session manager and session store. Will work on that.

Thanks.